packages/wdio-devtools-service/package.json

```diff
@@ -26,9 +26,11 @@
     "url": "https://github.com/webdriverio/webdriverio/issues"
   },
   "dependencies": {
-    "@types/puppeteer": "^5.4.0",
+    "@tracerbench/trace-event": "^4.5.2",
+    "@types/puppeteer-core": "^2.0.0",
```

Comment:
are these types mandatory as a dependency instead of a devDependency?

Reply:
Yes, as we are using them in our types.